fusion_report/parsers/starfusion_lr.py

```diff
@@ -7,10 +7,10 @@
 class Starfusion_lr(AbstractFusionTool):
```

Comment:
I'm confused about this. Is this [CTAT-LR-Fusion](https://github.com/TrinityCTAT/CTAT-LR-fusion)? If not add a link to the tool in the readme. If the same, I would rename to avoid confusion

Reply:
renamed